noty-android/app/src/main/java/dev/shreyaspatil/noty/view/viewmodel/NoteDetailViewModel.kt

```diff
@@ -119,6 +124,26 @@ class NoteDetailViewModel @AssistedInject constructor(
         }
     }
 
+    fun updatePin() {
```

Comment:
Can we rename this to `togglePin`? The word "toggle" I guess implies a better meaning than "update". Because the function whose name is "update" basically needs a param for updating. But in this case, we already know what we have to do. So "toggle" seems good to me. WDYT?

Reply:
Sounds good, I will do it.